src/components/Icon/index.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react'
+import { StyledIcon } from './styles'
+
+type Props = {
+  icon: string,
+  className: string,
+  width: string
```

Comment:
We could set a default width/height to `1em` and override this with `styled-components` where needed, nah? But if you’re happy with it, let’s do it this way.